cos_alerter/alerter.py

```diff
@@ -8,6 +8,7 @@
 import textwrap
 import threading
 import time
+from pathlib import Path
```

Comment:
technically since you're only using it for a type hint, you could make this a typing-time-only import (if you do it structurally, it saves some runtime and it avoids many issues with cyclic imports).

Reply:
or, to avoid importing path here and in daemon.py, you could do in THIS file the Path(str) casting and get rid of the import in daemon.py?

Reply:
Sure; I'll go with the second proposal for simplicity :)